src/components/Accordion/Accordion.types.ts

```diff
@@ -0,0 +1,21 @@
+import { ReactNode } from 'react';
+
+export type AccordionItem = {
+  title: string;
+  content: ReactNode;
+  isOpen?: boolean;
+  id: string | number;
```

Comment:
I would probably extract id type like `export type AccordionItemId = string | number` and then reuse it since from the implementation I see that id in AccordionItem and AccordionCollapsiblePros are the same. You can also use it here https://github.com/securityscorecard/design-system/pull/40/files#diff-a31ce76dd91c59f1f5ac8d8adbe78ee1ca5b3e195f546b8419f6f80554f6d626R18.